test/e2e/operator_expectations.go

```diff
@@ -121,3 +126,48 @@ func ExpectAllMachinesLinkedToANode() error {
 	}
 	return nil
 }
+
+func ExpectReconcileControllersDeployment() error {
+	key := types.NamespacedName{
+		Namespace: namespace,
+		Name:      "clusterapi-manager-controllers",
+	}
+	d := &kappsapi.Deployment{}
+
+	glog.Info("Get deployment")
+	err := wait.PollImmediate(1*time.Second, waitShort, func() (bool, error) {
+		if err := F.Client.Get(context.TODO(), key, d); err != nil {
```

Comment:
I would strongly suggest to rename `F` to `Framework` in the `test/e2e/main.go`. One letter variables are hard to find, especially when they are global.